pkg/node/peer_manager/storage/cbor.go

```diff
@@ -334,28 +436,50 @@ func (bs *CBORStorage) unsafeDropKnown() error {
 	return nil
 }
 
-func (bs *CBORStorage) unsafeSyncSuspended(newEntries, backup []SuspendedPeer) error {
-	if err := marshalToCborAndSyncToFile(bs.suspendedFilePath, bs.suspended); err != nil {
+func (bs *CBORStorage) restrictedFilePathByID(peersID restrictedPeersID) string {
+	switch peersID {
+	case suspendedPeersID:
+		return bs.suspendedFilePath
+	case blackListedPeersID:
+		return bs.blackListFilePath
+	default:
+		panic("unknown restricted id")
+	}
+}
+
+func restrictedNameByID(peersID restrictedPeersID) string {
+	switch peersID {
+	case suspendedPeersID:
+		return "suspended"
+	case blackListedPeersID:
+		return "blackList"
+	default:
+		panic("unknown restricted id")
+	}
+}
```

Comment:
I thins this function is also unnecessary. Add info about type in `AddToBlackList` and `AddSuspended` methods.